.eslintrc.yml

```diff
@@ -0,0 +1,11 @@
+env:
+  browser: true
+  es6: true
+extends: 'eslint:recommended'
+globals:
+  Atomics: readonly
+  SharedArrayBuffer: readonly
+parserOptions:
+  ecmaVersion: 2018
+  sourceType: module
+rules: {}
```

Comment:
- We need to make sure that our linters are setup the proper way, please do follow the instructions in the [Microverse linters repo for Javascript](https://github.com/microverseinc/linters-config/tree/master/javascript), and configure your eslint the way it is described there. Once that is done proceed with the linting process if necessary.